### ISSUE TITLE
Screen Readers are unable to track cursor in textarea

### ISSUE THREAD

**Issue by Flameborn** [Inscryb/inscryb-markdown-editor#18]
### I'm submitting a...
- [x] Bug report
- [ ] Feature request

### Reproduction steps

1. Type text into a MDE textarea, possibly with new lines.
2. While any screen reader is on, such as [NVDA](https://nvda-project.org) for Windows, VoiceOver on Mac or iOS, use the arrow keys to navigate around.

Result: The textarea is often detected as being empty, characters are not announced while the cursor moves.

Expected result: lines and characters are announced as the cursor moves.

### Version information
Browser type and version: any
SimpleMDE version: any

**Comment by CWharton** [comment]
This might be a codemirror issue. Will have to look into it.
